dev/com.ibm.ws.security.oidc.client_fat.jaxrs/fat/src/com/ibm/ws/security/openidconnect/client/fat/jaxrs/CommonTests/JaxRSClientReAuthnTests.java

```diff
@@ -42,6 +43,7 @@ public class JaxRSClientReAuthnTests extends CommonTest {
     public static HashMap<String, Integer> defRespStatusMap = null;
     protected static CommonCookieTools cookieTools = new CommonCookieTools();
     public static RSCommonTestTools rsTools = new RSCommonTestTools();
+    protected static int testsRun = 0;
```

Comment:
Could just be a boolean instead?

Reply:
yeah - started change with a different approach where the counter was needed - just pushed another commit.